src/test/java/org/prebid/server/ApplicationTest.java

```
@@ -1496,6 +1545,37 @@ private static String jsonFrom(String file) throws IOException {
                 ApplicationTest.class.getSimpleName() + "/" + file)));
     }
 
+    private static String cacheResponseFromRequestJson(String requestAsString, String requestCacheIdMapFile) {
+        ArrayNode responseArray = mapper.createArrayNode();
+
+        try {
+            JsonNode jsonNodeRequest = mapper.readTree(requestAsString);
+            JsonNode jsonNodeMatcher = mapper.readTree(ApplicationTest.class.getResourceAsStream(ApplicationTest.class.getSimpleName() + "/" + requestCacheIdMapFile));
+            ArrayNode puts = (ArrayNode) jsonNodeRequest.get("puts");
+
+            for (JsonNode putItem : puts) {
+                if (putItem.get("type").textValue().equals("json")) {
+                    String id = putItem.get("value").get("id").textValue() + "@" + putItem.get("value").get("price").doubleValue();
+                    String uuid = jsonNodeMatcher.get(id).textValue();
+                    responseArray.add(mapper.createObjectNode().put("uuid", uuid));
+                } else {
+                    String id = putItem.get("value").textValue();
+                    if(id == null) { //workaround for conversant
+                        id = "null";
+                    }
+                    String uuid = jsonNodeMatcher.get(id).textValue();
+                    responseArray.add( mapper.createObjectNode().put("uuid", uuid));
+                }
+            }
+
+            ObjectNode response = mapper.createObjectNode();
+            response.put("responses", responseArray);
+            return mapper.writeValueAsString(response);
+        } catch (IOException e) {
+            return mapper.createObjectNode().toString();
```

Maybe it's better to fail loudly instead of ignoring error?